.github/workflows/main.yml

```diff
@@ -201,8 +205,12 @@ jobs:
           python -m pip install tox
           tox --version
 
-      - name: Run tests
-        run: tox -- --ddtrace
+      - name: Run integration tests
+        uses: nick-fields/retry@v3
+        with:
+          timeout_minutes: 10
```

Comment:
Should this be 30 since the timeout for this job is 30 minutes?

Reply:
Good call, I doubled the max runtime of the last few runs (5 × 2 = 10 mins), but can set it to the timeout instead.